mantidproject/mantid
# Warn about division by 0 in NormaliseToMonitor.

Fixes #12891.

**To test**

Run this script:

```
import numpy as np

x = [1,2,3,4,5,6,7,8,9,10] * 3
y = [1,2,3,4,5,6,7,8,9,
       1,0,3,0,5,0,7,0,9,
       1,2,3,4,5,6,7,8,9]
e = np.sqrt(y)
ws = CreateWorkspace(x,y,e,3)

norm1 = NormaliseToMonitor(ws,MonitorSpectrum=1)
norm2 = NormaliseToMonitor(ws,MonitorSpectrum=2)
```

The second algorithm run should issue a warning about division by 0.
